include/realtime_tools/realtime_helpers.hpp

```diff
@@ -101,6 +115,17 @@ std::pair<bool, std::string> set_thread_affinity(std::thread & thread, int core)
 */
 std::pair<bool, std::string> set_current_thread_affinity(int core);
 
+/**
+ * Configure the current thread affinity - Tell the scheduler to prefer a certain
+ * core for the current thread.
+ * \note The threads created by the calling thread will inherit the affinity.
+ * \param[in] core the cpu numbers of the core. If an empty vector is passed,
```

Comment:
I think you missed updating the comments

Reply:
Well isn't it how it works. Previously it is -1 and now if the vector is empty should behave same

Reply:
Ah sorry I clicked in the wrong line:

> Tell the scheduler to prefer a certain
 * core for the current thread.
 
 This is what I meant. Should probably be: > Tell the scheduler to prefer a certain set of cores for the current thread